static/app/data/platformCategories.tsx

```diff
@@ -338,3 +338,6 @@ export const replayOnboardingPlatforms: readonly PlatformKey[] = [
   'javascript-vue',
   'javascript',
 ];
+
+// These are the supported platforms that can also be set up using the loader
+export const replayPlatformsLoader: readonly PlatformKey[] = ['javascript'];
```

Comment:
long term this will only include the BE platforms

Reply:
but also, this is a separate config outside of everything in `gettingStartedDocs` which could be annoying to maintain. 

it's also annoying to put `replayOnboardingJsLoader` in all the be files, so this is a tradeoff, but not the end of the world either way (easy to change back)